pallets/loans/src/lib.rs

```diff
@@ -116,7 +116,7 @@ pub mod pallet {
 
 	pub type PortfolioInfoOf<T> = Vec<(<T as Config>::LoanId, ActiveLoanInfo<T>)>;
 	pub type AssetOf<T> = (<T as Config>::CollectionId, <T as Config>::ItemId);
-	pub type PriceOf<T> = (<T as Config>::Balance, <T as Config>::Moment);
+	pub type PriceOf<T> = (<T as Config>::Balance, Seconds);
```

Comment:
When in `pallet_loans` we use `<T as Config>::Moment` is because we do not use seconds, or we do not know the time unit. In this case, this value comes from whatever the oracle configures for time, which is milliseconds.

Reply:
Yeah, but I collected it as seconds.